pkg/arm/data/master-startup.sh

```diff
@@ -562,6 +562,9 @@ EOF
 
 sed -i -re "s#( *server: ).*#\1https://$(hostname)#" /etc/origin/master/openshift-master.kubeconfig
 sed -i -re "s#( *server: ).*#\1https://$(hostname)#" /etc/origin/node/node.kubeconfig
+# HACK: copy node.kubeconfig to bootstrap.kubeconfig so that openshift start node used in the sync
+# daemonset will not fail and set the master node labels correctly.
+cp /etc/origin/node/node.kubeconfig /etc/origin/node/bootstrap.kubeconfig
```

Comment:
should we not be doing something like we do in the node startup:

```
base64 -d <<< {{ Base64Encode (YamlMarshal .Config.NodeBootstrapKubeconfig) }} >/etc/origin/node/bootstrap.kubeconfig
base64 -d <<< {{ Base64Encode (CertAsBytes .Config.NodeBootstrapCert) }} >/etc/origin/node/node-bootstrapper.crt
base64 -d <<< {{ Base64Encode (PrivateKeyAsBytes .Config.NodeBootstrapKey) }} >/etc/origin/node/node-bootstrapper.key
chmod 0600 /etc/origin/node/node-bootstrapper.key /etc/origin/node/bootstrap.kubeconfig
```

Reply:
Are these variables controlled by the `/etc/sysconfig/<origin|atomic-openshift>-node` file?  This might be another option.

Reply:
@pweil- we want to bootstrap master nodes using the admin kubeconfig (can't do otherwise because the api server runs as a static pod) and the bootstrap kubeconfig is not really used for anything when masters are bootstrapped (kubelet ignores bootstrap.kubeconfig when node.kubeconfig exists). Only `openshift start node --write-flags` breaks in the sync daemonset. Maybe deserves an upstream issue.